rnd/autogpt_server/autogpt_server/util/settings.py

```diff
@@ -95,6 +100,11 @@ class Config(UpdateTrackingModel["Config"], BaseSettings):
         description="The port for agent server daemon to run on",
     )
 
+    agent_api_port: int = Field(
+        default=8006,
+        description="The port for agent server API to run on",
+    )
+
```

Comment:
This seems unrelated, wasn't announced, isn't mentioned in the PR description, and just cost me half an hour of debugging.